### ISSUE TITLE
Improve README for Packet

### ISSUE THREAD

**Issue by vielmetti** [pharmer/pharmer#257]
Reference 

* https://github.com/appscode/pharmer/tree/master/cloud/providers/packet/
* https://github.com/appscode/pharmer/blob/master/cloud/providers/packet/README.md

## What I saw 

Packet is supported; README.md file is minimal. Thanks for your support!

## What I hoped to see

More information about Packet as a provider, including a signup link and a link to Packet documentation.

## What I hope to do

Make this improvement in a PR after testing it.

**Comment by tamalsaha** [comment]
@vielmetti, thanks for the input. Please send a pr. 

We also hope to write a more detailed tutorial on how to setup a packet cluster similar to https://github.com/appscode/pharmer/blob/master/cloud/providers/digitalocean/README.md doc.

**Comment by vielmetti** [comment]
Thanks @tamalsaha , provisioning my first test cluster now. I opened a ticket #263 re spot market, since that can be an economical way to test a cluster (you'd have to be very careful to run it in production, but for tests it's ideal).

**Comment by vielmetti** [comment]
First try at provisioning timed out with

```
I1121 11:03:18.727588   25925 apiserver.go:67] Attempt 182: Probing Kubernetes api server ...
Get https://147.75.79.225:6443/api/v1/pods: dial tcp 147.75.79.225:6443: getsockopt: connection refused ,.,.,.,.,.,
F1121 11:03:18.773018   25925 apply.go:38] timed out waiting for the condition
```

**Comment by vielmetti** [comment]
Provisioning succeeded, kubectl recognizes the cluster. Working on `ssh`.

**Comment by vielmetti** [comment]
`ssh` working after https://github.com/appscode/pharmer/pull/270

**Comment by vielmetti** [comment]
`spot-market` ready to test.

**Comment by vielmetti** [comment]
Closed by #305 thanks!
